modules/test/playwright/pages/segments-web/SegmentEditorPage.ts

```diff
@@ -50,19 +55,29 @@ export class SegmentEditorPage {
 
 		for (const [section, items] of Object.entries(properties)) {
 			for (const property of items) {
-				await this.addProperty(section as SegmentSection, property);
+				await this.addProperty(
+					configure,
+					section as SegmentSection,
+					property
+				);
 			}
 		}
 
-		await this.saveButton.click();
+		if (submit) {
+			await this.saveButton.click();
+		}
 	}
 
 	/**
 	 * Adds the given property in the last position
 	 * of the given section
 	 */
 
-	async addProperty(section: SegmentSection, property: SegmentProperty) {
+	async addProperty(
+		configure: boolean,
```

Comment:
also here, if we put configure at the end we can give it a default value so we don't need to pass always a value